.github/workflows/push-images.yml

```
@@ -0,0 +1,43 @@
+name: Push-Images
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+          
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          
+      - name: Login to GitHub Container Registry
+        run: echo $CR_PAT | docker login ghcr.io -u USERNAME --password-stdin
+        env:
+          CR_PAT: ${{ secrets.CR_PAT }}
```

```suggestion
      - ame: Login to GitHub Container Registry
        uses: docker/login-action@v2
        with:
          registry: ghcr.io
          username: ${{ github.actor }}
          password: ${{ secrets.GITHUB_TOKEN }}
```

We can use `docker/login-action` here.